core/src/main/java/com/dtolabs/rundeck/core/audit/ResourceTypes.java

```diff
@@ -0,0 +1,15 @@
+package com.dtolabs.rundeck.core.audit;
+
+/**
+ * Types of resources reported by audit events.
+ */
+public final class ResourceTypes {
```

Comment:
This seems like it should be an enum

Reply:
This was indeed an enum. However i changed it to be strings so the interface could support adding more resource and action types dynamically without modiying the actual code, so we could implement generation of dynamic or configurable resource types or actions in the future.

Reply:
Ok, that makes sense